spec/ruby/core/exception/system_call_error_spec.rb

```diff
@@ -25,6 +25,7 @@ def initialize
     @example_errno_class = Errno::EINVAL
     @last_known_errno = Errno.constants.size - 1
     @unknown_errno = Errno.constants.size
+    @some_human_readable = "[[:graph:]]+"
```

Comment:
Shouldn't this be a regular expression, as the same as spec/ruby/optional/capi/kernel_spec.rb?
```suggestion
    @some_human_readable = /[[:graph:]]+/
```